### DISCUSSION TITLE
N notes found not printed to stdout?

### DISCUSSION THREAD

**Discussion by mhanberg** [zk-org/zk#130] [Help] [answered]
I've noticed that when you pipe the output of a command into another command, the "22 notes found" text still prints afterwards.

Is this intentional? Is it going to stderr?

<img width="255" alt="image" src="https://user-images.githubusercontent.com/5523984/146691321-1f4bf32c-d385-4abf-8ba3-6930b58cdfac.png">

**Comment by mickael-menu** [comment, marked as answer]
Right, it's printed to `stderr` to allow post-processing of the output of `zk list`.

It's mostly useful combined with `--format`, for example:

```
zk list --format path | xargs ls -l
```

You can add `--quiet` to disable printing out the count altogether.
